docs/react/guides/installing-tailwindcss-in-web-react-nextjs-apps.md

```diff
@@ -0,0 +1,71 @@
+# Installing TailwindCss in Web/React/Next.js Apps
```

Comment:
Should name the guide `Using Tailwind CSS in React`. We can mention that it is for both Next.js and React in the details. Not need to mention Web.